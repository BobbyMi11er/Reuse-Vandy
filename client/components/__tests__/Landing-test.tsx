import * as React from 'react';
import renderer from 'react-test-renderer';

import LandingPage from '@/app/(auth)/landing';

it(`renders correctly`, () => {
  const tree = renderer.create(<LandingPage/>).toJSON();

  expect(tree).toMatchSnapshot();
});
