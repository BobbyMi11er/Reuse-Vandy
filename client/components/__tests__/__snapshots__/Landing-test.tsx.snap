// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#F8F8F8",
      "flex": 1,
      "justifyContent": "space-between",
    }
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        {
          "color": "#000",
          "fontSize": 24,
          "fontWeight": "bold",
        }
      }
    >
      Reuse Vandy
    </Text>
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "backgroundColor": "#F4A71D",
          "borderRadius": 5,
          "marginTop": 20,
          "opacity": 1,
          "paddingHorizontal": 30,
          "paddingVertical": 10,
        }
      }
    >
      <Text
        style={
          {
            "color": "#000",
            "fontSize": 16,
            "fontWeight": "bold",
          }
        }
      >
        Sign in
      </Text>
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "center",
          "marginTop": 20,
        }
      }
    >
      <Text
        style={
          {
            "color": "#888",
            "fontSize": 12,
            "marginBottom": 20,
          }
        }
      >
        Don't have an account?
         
        <Text
          onPress={[Function]}
          style={
            {
              "color": "#F4A71D",
              "fontSize": 12,
              "fontWeight": "bold",
            }
          }
        >
          Sign up
        </Text>
      </Text>
    </View>
  </View>
  <Text
    style={
      {
        "color": "#888",
        "fontSize": 12,
        "marginBottom": 20,
      }
    }
  >
    Copyright Â© SWE Group 15 All Rights Reserved
  </Text>
</RCTSafeAreaView>
`;
